tests/scripts/common_functions.sh

```diff
@@ -135,6 +139,8 @@ function deploy_autotune() {
 	echo "Deploying autotune"
 	if [ -z "${AUTOTUNE_IMAGE}" ]; then
 		cmd="./deploy.sh -c ${cluster_type}"
+	elif [[ ! -z "${AUTOTUNE_IMAGE}" && ! -z "${CONFIG_DIR}" ]]; then
+		cmd="./deploy.sh -c ${cluster_type} -i ${AUTOTUNE_IMAGE} -d ${CONFIG_DIR}"
 	else
 		cmd="./deploy.sh -c ${cluster_type} -i ${AUTOTUNE_IMAGE}"
```

Comment:
In the else we will need to pass the config map by changing just the logging level to debug.